server.py

```diff
@@ -34,13 +34,20 @@
 from utils import url_fails
 from utils import get_video_duration
 
-from settings import settings, DEFAULTS
 from werkzeug.wrappers import Request
+
+from settings import DEFAULTS
+from settings import config_dir
+from settings import config_file
+from settings import load as load_settings
+from settings import save as save_settings
+
```

Comment:
load_settings it's just a function, which can load settings. There is no silently actions on module load))